arenadata/Dockerfile

```diff
@@ -3,6 +3,10 @@ FROM centos:centos7 as base
 ARG sigar=https://downloads.adsw.io/ADB/6.22.0_arenadata38/centos/7/community/x86_64/sigar-1.6.5-1056.git2932df5.el7.x86_64.rpm
 ARG sigar_headers=http://downloads.adsw.io/ADB/6.22.0_arenadata38/centos/7/community/x86_64/sigar-headers-1.6.5-1056.git2932df5.el7.x86_64.rpm
 
+# Download langpacks
+RUN sed -i 's/\(override_install_langs*\)/# \1/' /etc/yum.conf
+RUN yum -y reinstall glibc-common
```

Comment:
Please describe why we need to reinstall this package, combine steps to one (each Dockerfile command including RUN create a new layer), clean cache after reinstall.

Reply:
Added description and combined into one command